### ISSUE TITLE
dilatation of intrahepatic biliary ducts

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6122]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.cell.com/action/showPdf?pii=S0002-9297%2820%2930114-2

Patient 1/6

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
